lib/geocoder/lookups/default_lookup_methods.rb

```diff
@@ -0,0 +1,43 @@
+module Geocoder::Lookup
+  module DefaultLookupMethods
```

Comment:
@arpit1094 This file is `here` maps related. The module name/namespace should suggest that.